Sources/SwiftQueue/SqOperationQueue.swift

```diff
@@ -136,6 +136,14 @@ internal final class SqOperationQueue: OperationQueue {
             operation.cancel()
         }
     }
+    
+    func operationCount(tag: String) -> Int {
+        return operations.compactMap({ $0 as? SqOperation }).filter({ $0.info.tags.contains(tag) }).count
+    }
+    
+    func operationCount(uuid: String) -> Int {
```

Comment:
Job ID is supposed to be unique right ? So I don't get the point of returning a Int. Shouldn't be boolean ?

Reply:
I did not check all the code, but is that uniqueness enforced? 
or is it just supposed to be unique?
Nevertheless it should be bool!

Reply:
There is a unique UUID constraint per queue to enforce this

Reply:
I see...
Ok. I've updated the PR